pkg/agent/openflow/client.go

```diff
@@ -352,6 +371,9 @@ func (c *client) InstallNodeFlows(hostname string,
 			// tunnelPeerIP is the Node Internal Address. In a dual-stack setup, whether this address is an IPv4 address or an
 			// IPv6 one is decided by the address family of Node Internal Address.
 			flows = append(flows, c.l3FwdFlowToRemote(localGatewayMAC, *peerPodCIDR, tunnelPeerIP, cookie.Node))
+		} else if runtime.IsWindowsPlatform() && !c.encapMode.NeedsRoutingToPeer(tunnelPeerIP, c.nodeConfig.NodeIPAddr) && remoteGatewayMAC != nil {
+			flows = append(flows, c.l3FwdFlowToRemoteViaRouting(remoteGatewayMAC, *peerPodCIDR, cookie.Node)...)
+			modify = true
```

Comment:
If no flows were installed before for this Node, would setting it to modify have problem?

Can modify remove extra flows if the old flows have more flows than new ones? This is not true when it changes from `l3FwdFlowToRemote` to `l3FwdFlowToRemoteViaRouting`, but the `modifyFlows` method as a generic function should take care of it, otherwise it might cause obscure issues in the future.

Reply:
Thanks. I have changed `modifyFlows` to `hardModifyFlows` and refactored those operation methods in `pipeline.go`. It means that flows will be exactly as the input here.